apps/meteor/ee/app/license/server/airGappedRestrictionsCheck.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { License, AirGappedRestriction } from '@rocket.chat/license';
+import { Statistics } from '@rocket.chat/models';
+
+import { checkAirGappedRestrictions } from './airGappedRestrictionsCheck';
+
+jest.mock('@rocket.chat/models', () => ({
```

Comment:
Jest is not working/available in this context

Reply:
For server tests we use mocha + proxyquire. Not sure if we are moving towards using jest but for now i'd suggest to rewrite the tests using the tooling of the other server tests

Reply:
This review was outdated. I already fixed it using proxyquire